editor/export/editor_export_plugin.h

```diff
@@ -157,6 +158,7 @@ class EditorExportPlugin : public RefCounted {
 	virtual void _get_export_options(const Ref<EditorExportPlatform> &p_export_platform, List<EditorExportPlatform::ExportOption> *r_options) const;
 	virtual bool _should_update_export_options(const Ref<EditorExportPlatform> &p_export_platform) const;
 	virtual String _get_export_option_warning(const Ref<EditorExportPlatform> &p_export_platform, const String &p_option_name) const;
+	virtual void _get_export_option_overrides(const Ref<EditorExportPlatform> &p_export_platform, HashMap<StringName, Variant> *r_value_overrides) const;
```

Comment:
nit: should it be `_get_export_options_overrides` as we're providing overrides for multiple options instead of a single one?

Reply:
Hm. Well, `_get_export_option_overrides()` is what I would say outloud, and the "overrides" part is already plural which implies that it's more than a single one.

But I do see what you mean - Godot's APIs frequently build up on the name of a "base method". We have `_get_export_options()` and so adding `_overrides()` to that and arriving at `_get_export_options_overrides()` makes sense.

I could go either way :-)

Reply:
Alright! Changed in my latest push